src/qml.h

```diff
@@ -0,0 +1,30 @@
+// library includes
+#include <boost/filesystem.hpp>
+#include <linuxdeploy/core/appdir.h>
+
+#pragma once
+
+typedef struct {
+    std::string name;
+    boost::filesystem::path path;
+    boost::filesystem::path relativePath;
+} QmlModuleImport;
+
+static const char* const ENV_KEY_QML_MODULES_PATHS = "LINUXDEPLOY_QT_PLUGIN_QML_MODULES_PATHS";
```

Comment:
We should remove the `LINUXDEPLOY_QT_PLUGIN_` prefix here. These env var titles are just way too long.

Reply:
I used the prefix to avoid name clashing whit other variables set by the user. But let's hope that it will not happen.

Reply:
I am 99.9% sure it won't be a problem.